VEHICLE/SetVehicleMaxSpeed.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: VEHICLE
+aliases: ["0xBAA045B4E42F3C06"]
+---
+## _SET_VEHICLE_MAX_SPEED
+
+```c
+// 0xBAA045B4E42F3C06
+void _SET_VEHICLE_MAX_SPEED(Vehicle vehicle, float speed);
+```
+
+```
```

Comment:
Please remove the code block from the description. It's a descriptive message about the native, it's not a code snippet ;)

Reply:
Done!